File: CURRENT/hello-world/4-cpp-cli-visual-studio-2015/HelloWorld/HelloWorld.cpp
// HelloWorld.cpp : main project file.

#include "stdafx.h"

using namespace System;
using namespace org::mariuszgromada::math::mxparser;

int main(array<System::String ^> ^args)
{
	Expression ^eh = gcnew Expression("5^2 * 7^3 * 11^1 * 67^1 * 49201^1");
	Expression ^ew = gcnew Expression("71^1 * 218549^1 * 6195547^1");
	String ^h = mXparser::numberToAsciiString(eh->calculate());
	String ^w = mXparser::numberToAsciiString(ew->calculate());
	mXparser::consolePrintln(h + " " + w);
	return 0;
}

File: STABLE/2.1.1-1/hello-world/3-visual-basic-visual-studio-2015/HelloWorld/HelloWorld.vb
Imports org.mariuszgromada.math.mxparser

Namespace mxparserhelloworld
    Module HelloWorld
        Sub Main()
            Dim eh As Expression = New Expression("5^2 * 7^3 * 11^1 * 67^1 * 49201^1")
            Dim ew As Expression = New Expression("71^1 * 218549^1 * 6195547^1")
            Dim h As String = mXparser.numberToAsciiString(eh.calculate())
            Dim w As String = mXparser.numberToAsciiString(ew.calculate())
            mXparser.consolePrintln(h + " " + w)
        End Sub
    End Module
End Namespace
